File: BreadcrumbUI/Breadcrumb/src/app/Models/tbFilesDataModel.ts
export interface tbFilesDataModel {
    id: string;
    description: string;
    type: string;
    name: string;
    thumbnailLink: string;
}

export interface FilesWithShowToInsertModel {
    id: string;
    description: string;
    type: string;
    name: string;
    thumbnailLink: string;
    quality: string;
    audioLanguages:string;
    subtitleLanguages:string;
    tbShowsFileID: string | null;
}

File: BreadcrumbAPI/Breadcrumb.Model/FilesModels/tbFilesDataModel.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Breadcrumb.Model.FilesModels
{
    public class tbFilesDataModel
    {
        public Guid Id { get; set; }
        public string Description { get; set; }
        public string Type { get; set; }
        public string Name { get; set; }
        public string ThumbnailLink { get; set; }
    }
}
